### PR TITLE
fix(fuel): Lowered C/D spawn fuel, fixed center tank refueling 

### LINKED ISSUE #8092: Lower starting fuel amount
### Aircraft Version

Experimental

### Description

currently the aircraft loads in with around 9000kg of fuel in cold and dark which is very unrealistic and renders gsx fueling integration almost useless because for many flights you would have to defuel..

It would be far more realistic to have this around 2-3t of fuel

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

_No response_

### PR DESCRIPTION
**Pull request by Maximilian-Reuter**
Fixes #8092

## Summary of Changes

This PR aims to lower the fuel amount on C/D spawns to 2.5 tons. This way it reflects reality a bit better and almost always requires to refuel before flight.

Additionally I also changed the logic behind loading the saved fuel from last shutdown. Previously it would always be loaded no matter where one spawns. To me that didn't make sense as If someone wants to spawn on the runway or in the air, they are obviously forgoing refueling and therefore need a reasonable amount of fuel in the tanks. Currently that is 50% of the fuel load. 

This can be changed by adding tank levels to the FLT files if desired.

Finally fixed the longstanding issue where our plane would only fuel the center tanks after the wing tanks had been filled. In reality both center and wing tanks get filled at the same time.

## Additional context

Discord username (if different from GitHub):\_chaoz_

## Testing instructions

First make sure you are using the default FBW livery and that the A32NX.ini file in the location described here is deleted:
https://docs.flybywiresim.com/fbw-a32nx/feature-guides/loading-fuel-weight/?h=aircraftstate#fuel-persistence

1. Spawn C/D
2. Check fuel amount, should be 2.5 tons in only the inner tanks
3. Fill up fuel tanks completely and observe center tanks getting filled while wing tanks are filling
4. Detank fuel and observe the same in reverse
5. Spawn on runway
6. Check fuel amount to be 50% 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

**Comment by 2hwk** [comment]
Approved but we probably need to test this a bit to see what the impacts/implications will be

**Comment by Popespice** [comment]
Quality Assurance Report

Discord          : Popespice
Object of testing:#8502
Tier of Testing  : 2
Date             : 23022024

Testing Process: 
1. Go into work folder and delete saved aircraft state
2. Load into game and spawn in at any airport
3. Power on plane as usual
4. Check flypad, observe plane has 2.5 tons (kilograms) of fuel onboard
5. Fill tanks to max
6. Observe wing and center tanks fuel concurrently
7. Drain tanks back to 2.5 tons
8. Observe tanks defuel concurrently
9. Spawn back in direct to runway
10. Observe fuel on flypad is at 50% marker by default

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
Working as intended

Media:       
N/A

**Comment by Maximilian-Reuter** [comment]
Waiting for @2hwk to give okay

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -70,6 +70,7 @@
 1. [EFCS/AP/ATHR] Improved support of Active Pause - @aguther (Andreas Guther)
 1. [EFB] Updated taxi, landing, parking & secure checklists to november 2023 revision - @BravoMike99 (bruno_pt99)
 1. [ND/TCAS] Fixed disappearing TCAS targets - @ArtemisPlayer (ArtemisPlayer)
+1. [Fuel] Lowered starting fuel on C/D spawn, will only load last saved fuel on C/D spawn, center tank refuel now happens simultaneous with wing refuel - @Maximilian-Reuter (\_Chaoz_)
```
Comment (2hwk) [inline .github/CHANGELOG.md:73]:
```suggestion
1. [FUEL] Lowered starting fuel on C/D spawn, will only load last saved fuel on C/D spawn, center tank refuel now happens simultaneous with wing refuel - @Maximilian-Reuter (\_Chaoz_)
```
Fuel in caps please to match the format (I could commit this myself if needed)

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Comment by 2hwk** [comment]
QA Report

- [x] Spawn C/D
- [x] Check fuel amount, should be 2.5 tons in only the inner tanks
- [x] Fill up fuel tanks completely and observe center tanks getting filled while wing tanks are filling
- [x] Detank fuel and observe the same in reverse
- [x] Spawn on runway
- [x] Check fuel amount to be 50% 

My own checks: 

- [ ] Check for slider init value matching the initial spawn fuel value - Failed

image

Should be:

image

Just to confirm that it's what I suspect is going on:

image

image


Looks like the line of JS that initialises the desired fuel level that controls the slider (After READY TO FLY screen) is loaded BEFORE the FADEC WASM code sets the fuel, which means the wrong fuel amount is initialised on C&D spawn.

The slider should reflect the starting spawn amount of fuel when you first load in (Yes I know this UX is not perfect or 100% intuitive, but it is what it is) 

Result:
 - QA Not Passed

**Pull request merged**